### PR TITLE
init data retention command

### PR DESCRIPTION
**Pull request by pivnicek**
re: https://github.com/keboola/connection/issues/1040

requires: https://github.com/keboola/storage-api-php-client/pull/158

### CODE REVIEW

**Comment by pivnicek** [comment]
Can probably update this to use the manage API to get the projects, rather than feeding a list. https://keboolamanagementapi.docs.apiary.io/#reference/maintainers/maintainers/list-maintainers

**Comment by Halama** [comment]
I think it is ok with feeding a list. It is more flexible, you can try it first with few projects. Also restart from last project in case of some failure.

File: src/Keboola/Console/Command/InitDataRetention.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        $fh = fopen('php://stdin', 'r');
+        if (!$fh) {
+            throw new \Exception('Error on input read');
+        }
+
+        $lineNumber = 0;
+        while ($row = fgetcsv($fh)) {
+            if ($lineNumber === 0) {
+                $this->validateHeader($row);
```
Comment (Halama) [inline src/Keboola/Console/Command/InitDataRetention.php]: This is doing nothing :)

Comment (Halama) [reply src/Keboola/Console/Command/InitDataRetention.php]: since that method does not throw exception

Comment (pivnicek) [reply src/Keboola/Console/Command/InitDataRetention.php]: omg :-) diky!

**Pull request merged**